TodoList/Controllers/LogsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Win32.SafeHandles;
+using TodoList.DTOs;
+using TodoList.Models;
+using TodoList.Services.Log;
+using TodoList.Services.PasswordHash;
+using TodoList.Services.TokenGenerator;
+
+namespace TodoList.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthsController : ControllerBase
+    {
+        private readonly ILog _ilog;
```

Comment:
```suggestion
        private readonly IUserService _userService;
```
Đổi tên file ILog và Log